src/list.cpp

```diff
@@ -286,6 +287,8 @@ void list_probes(const BPFtrace &bpftrace, const std::string &search_input)
     std::cout << probe << std::endl;
   }
 
+  // kfuncs
+  bpftrace.btf_.display_funcs();
```

Comment:
All other list functions support regex based filtering. e.g. `bpftrace -l 'k:get*'`, can you add that here too? (see `search_probe()`)